src/components/LogList.jsx

```diff
@@ -8,36 +8,35 @@ function LogList() {
   const [query, setQuery] = useState('');
 
   const getAllLogs = async () => {
-    try{
-      const response = await axios.get('http://localhost:3300/apiv1/log/getAllLogs');
+    try {
+      const response = await axios.get(
+        'http://localhost:3300/apiv1/log/getAllLogs'
```

Comment:
Can we carry this part: `http://localhost:3300/` to the env file and store it in a variable in the component and store all the moving parts of the URL in a constants object in a file called constants. We can create a utils folder for those things.